supervisely/api/api.py

```diff
@@ -1504,17 +1520,19 @@ async def stream_async(
             request=resp.request if locals().get("resp") else None,
         )
 
-    def _set_async_client(self):
+    async def _set_async_client(self):
         """
         Set async httpx client if it is not set yet.
         Switch on HTTP/2 if the server address uses HTTPS.
         """
         if self.async_httpx_client is None:
             self._check_https_redirect()
+            http2 = False
             if self.server_address.startswith("https://"):
-                self.async_httpx_client = httpx.AsyncClient(http2=True)
-            else:
-                self.async_httpx_client = httpx.AsyncClient()
+                if self._http2_supported is None:
+                    await self._verify_http2_support_status_async()
+                http2 = self._http2_supported
+            self.async_httpx_client = httpx.AsyncClient(http2=http2)
```

Comment:
Why async?

Reply:
Because it was only running in an async context, it would have been better to use `await` for `self._verify_http2_support_status_async()`. However, since these methods have now been removed, this is no longer relevant.